### DISCUSSION TITLE
CSV Import Failing

### DISCUSSION THREAD

**Discussion by becausebirds** [plausible/analytics#4171] [Self-Hosted Support] [answered]
I'm experiencing an error with the CSV import feature. Looking at the docker logs it appears to trip up when running the import statement.

`<Error> executeQuery: Code: 73. DB::Exception: Unknown format CSVWithNamesdate.`

Attached is an excerpt of the error log and an example of how one of the csv import files is structured. Any advice is greatly appreciated!

edit: *removed*

**Comment by ruslandoga** [comment, marked as answer]
@becausebirds 👋 

Thank you for reporting it! I think https://github.com/plausible/analytics/pull/4172 would fix it.

**Reply by ruslandoga** [reply]
[v2.1.1](https://github.com/plausible/analytics/discussions/4195) has been released with this fix.
